bin/process-inactive-accounts

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env perl
+
+use v5.14;
+use warnings;
+
+BEGIN {
+    use File::Basename qw(dirname);
+    use File::Spec;
+    my $d = dirname(File::Spec->rel2abs($0));
+    require "$d/../setenv.pl";
+}
+
+use Getopt::Long;
+use FixMyStreet::Script::Inactive;
+use Pod::Usage;
+
+my %h;
+GetOptions(\%h, 'anonymize=i', 'email=i', 'verbose|v', 'help|h');
+pod2usage(0) if $h{help};
+pod2usage(1) unless $h{anonymize} && $h{email};
+pod2usage("Anonymize time must be greater than email time")
```

Comment:
I wonder if being able to only anonymize accounts without email some would be useful?

Reply:
Yes, good idea.